sdk/src/main/java/com/uber/sdk/rides/client/Session.java

```diff
@@ -96,6 +100,15 @@ public Builder setServerToken(String serverToken) {
             this.serverToken = serverToken;
             return this;
         }
+        
+        /**
+         * Sets the requested locale through the Accept-Language http header. See https://developer.uber.com/docs/localization for
```

Comment:
Nit: End sentence with a period, linkify the URL, capitalize HTTP, lowercase Locales.

"Sets locale in the Accept-Language HTTP header. See the <a href="https://developer.uber.com/docs/localization">Uber API Localization documentation</a> for possible locales."